cms/envs/devstack.py

```diff
@@ -117,14 +117,6 @@ def should_show_debug_toolbar(request):
 ################################ MILESTONES ################################
 FEATURES['MILESTONES_APP'] = True
 
-########################### ORGANIZATIONS #################################
```

Comment:
[Curious about best practices] Is it the case that if we don't want devstack.py to override something in common.py, we leave it out of the file?  Or should `ORGANIZATIONS_AUTOCREATE` be in here even if it's `True` so folks don't have to do a mental merge of the various config files?

Reply:
@davidjoy Great question; I see both patterns used, so I'm not sure we have a best practice. I think the real "best practice" is what we decided upon in [OEP-45](https://open-edx-proposals.readthedocs.io/en/latest/oep-0045-arch-ops-and-config.html) -- that is, flat, declarative (YAML) config files, instead of the layered Python settings we have here. But edx-platform isn't there yet.

I'd originally added this redundant block, and was conflicted on whether to modify it or just rip it out for this PR. In this case, I'm leaning towards adding it back--even though it's the default to in `common.py`, it's notably different than what we configure in Production, so the explicitness might be helpful to something down the road.

Reply:
@davidjoy Added back the no-op override, with a new comment.